components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -704,13 +707,54 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
                resolution = sqrt(c_mode1 / (fEdge(iEdge)**2.0 + betaEdge(iEdge)*c_mode1 + 1.0E-15_RKIND)) &
                             / dcEdge(iEdge)
                if (resolution .ge. 2.0 .or. c_mode1 .lt. 0.1_RKIND) then
-                  gmResolutionTaper(iEdge) = 0.0_RKIND
+                  RossbyRadiusTaper(iEdge) = 0.0_RKIND
                else
-                  gmResolutionTaper(iEdge) = 1.0_RKIND
+                  RossbyRadiusTaper(iEdge) = 1.0_RKIND
                end if
             end do
             !$omp end do
             !$omp end parallel
+
+            if (config_GM_horizontal_taper == 'RossbyRadius') then
+               !$omp parallel
+               !$omp do schedule(runtime)
+               do iEdge = 1,nEdges
+                  gmHorizontalTaper(iEdge) = RossbyRadiusTaper(iEdge)
+               end do
+               !$omp end do
+               !$omp end parallel
+            end if
+
+            if (config_Redi_horizontal_taper == 'RossbyRadius') then
+               !$omp parallel
+               !$omp do schedule(runtime)
+               do iEdge = 1,nEdges
+                  RediHorizontalTaper(iEdge) = RossbyRadiusTaper(iEdge)
+               end do
+               !$omp end do
+               !$omp end parallel
+            end if
+
+            deallocate (RossbyRadiusTaper)
+         end if
+
+         if (config_Redi_closure == 'equalGM') then
```

Comment:
Need to update the Registry for `config_Redi_closure`. (From what I understand PR #4867 is being included in here)